README.md

```diff
@@ -6,97 +6,128 @@ The goal of this project is to empower citizens to effectively organize for thei
 
 ## Getting Started
 
-The project is in development and has two parts:
+This project is under development and has three main parts - backend, frontend and crawler.
 
 ## Setup instructions for development
 
-### Prerequisites for backend
+### Prerequisites
 
 Things you need to install:
 
-* git
-* Nodejs
-* Mysql
+* Git
+* Node.js (we currently support release 8.x)
+* MySQL (required only for the backend & crawler)
 
-Also these requirements:
+Once you have these you can download the code and install dependencies:
 
-https://github.com/nodejs/node-gyp#installation
+```bash
+$ git clone git@github.com:meirim-org/meirim.git
+$ cd meirim/
+$ npm install
+```
 
 ### Instructions for backend
 
-Download code and dependencies
+Connect to your MySQL instance:
 
 ```bash
-npm install knex -g
-git clone git@github.com:meirim-org/meirim.git
-cd meirim/
-npm i
+$ mysql -uroot -p
 ```
 
-Install chrome dependencies (for ubuntu):
+Create a database for the project:
 
-```bash
- apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
- ```
+```sql
+CREATE DATABASE meirim character set UTF8 collate utf8_bin;
+exit;
+```
 
-Import the database schema
+Edit the local configuration file and set your database and email settings (if needed):
```

Comment:
I think that it's best if we will write specifically what configurations should be changed inside this files under what terms. (db password? db user? etc)

Reply:
changed it a bit to list the name of the file. if you view the file you will see which exact configuration values should be changed